plugins/node/opentelemetry-instrumentation-ioredis/src/instrumentation.ts

```diff
@@ -48,33 +48,33 @@ export class IORedisInstrumentation extends InstrumentationBase<
   init(): InstrumentationNodeModuleDefinition<typeof ioredisTypes>[] {
     return [
       new InstrumentationNodeModuleDefinition<typeof ioredisTypes>(
-        'ioredis',
-        ['>1 <5'],
```

Comment:
we can't drop support for older versions i would advise to make multiple `InstrumentationNodeModuleDefinition`, one for each versions

Reply:
Since the types are not compatible, I don't see how both can be supported in the same plugin. I can create a `opentelemetry-instrumentation-ioredis-4` with the previous version of the plugin?

Reply:
That would be breaking for end-users, i think its possible to keep them both here if you keep manually typed v4 types that was there WDYT ?